src/azure-cli-core/azure/cli/core/commands/progress.py

```diff
@@ -114,7 +114,7 @@ def write(self, args):
         :param args: dictionary containing key 'message'
         """
         if self.spinner is None:
-            self.spinner = humanfriendly.Spinner(
+            self.spinner = humanfriendly.Spinner(  # pylint: disable=no-member
```

Comment:
`humanfriendly.Spinner` is moved to `humanfriendly.terminal.spinners.Spinner` in later versions. Use the deprecated `humanfriendly.Spinner` for compatibility.